### DISCUSSION TITLE
Is kFixedChannels only available on arch >= SM80?

### DISCUSSION THREAD

**Discussion by lygztq** [NVIDIA/cutlass#538] [Q&A] [answered]
Hi, I'm studying the first layer Convolution kernels mentioned [here](https://github.com/NVIDIA/cutlass/pull/468). And I find it is impossible for me to generate conv instances using the python interface on V100. So, just to be sure, are these first layer convolution kernels only available on arch >= SM80?

**Comment by hwu36** [comment, marked as answer]
sm75 and sm80 are tested, but i think sm70 should be supported out of the box.

our `generator.py` does not have sm70, but you can use this one https://github.com/NVIDIA/cutlass/blob/master/tools/library/scripts/generator.py#L478 as a boilerplate and uses the tile sizes and instruction sizes in https://github.com/NVIDIA/cutlass/blob/master/tools/library/scripts/generator.py#L819 instead.

**Reply by lygztq** [reply]
Thanks for your answer! That helps a lot.
